lib/attack.go

```diff
@@ -222,11 +234,26 @@ func (r Rate) IsZero() bool {
 	return r.Freq == 0 || r.Per == 0
 }
 
+// Interval returns the time between now and the next hit.
```

Comment:
Don't you think that docs for `Rater.Interval` and `Rate.Interval` should be that same ? 

I mean, reading both `Interval returns the time the attacker needs to sleep before the next hit is sent to the target.` and `Interval returns the time between now and the next hit.` it is not quite clear that they are equivalent (and if they are, it would be better to just put it in one place - just document interface)

Reply:
They are approximately the same because Rate implements Rater. I can nuke this comment if you want, but then `go vet` will then complain about an exported function that is undocumented, and your documentation will be incomplete.

Other options:

  - Make the comments *exactly* the same, rather than just mostly the same.
  - Change the comment to read something like:

        Interval implements part of the Rater interface. It calculates the time between each hit
        from `r.Per / r.Freq` and multiplies this by the count of hits already elapsed to determine
        the absolute time when the next hit should occur. It returns the Duration until that time.

LMK what you'd prefer I do?